ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+declare -A external_network=(
```

Comment:
2 comments (that can be addressed later and not necessarily in this PR, just keep it in mind please):


* I know we're constrained by bash here but I wish we could make it so it's kind of like a dictionary here, where we define something like

```{ <cloud-name> : { network: external, flavor: xlarge } }```

* I wish we could name the clouds a bit better. Instead of "openstack", name it "MOC", etc.

Reply:
comment 1:
We could go fancier indeed. I initially wrote the selection logic as a bash-subtitution oneliner, but it was unreadable. I hope to have stroke an acceptable balance between functionality/readability/complexity; we can always do better in a follow-up indeed.

comment 2:
The clouds are oddly named, I agree. We can change that in a follow-up once we find a better naming. That will require updates to:
* lease naming
* ideally, secret names in our vault for consistency
* secret mirroring
* and finally, this `openstack-conf-resourcenames`.